qchem/tests/test_excitations.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
Do we need the usual copyright clause?

Reply:
This is a good point. I have never added it but I noticed it is in the unit test of PennyLane. Should I add it also to `qchem` tests?

Reply:
May as well!